service/src/main/java/bio/terra/pipelines/dependencies/gcs/GcsClient.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.pipelines.dependencies.gcs;
+
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GcsClient {
```

Comment:
wasn't totally sure what should go into the client vs service on this one

Reply:
i think having this client is a good separation.  this should be simple and have no business logic in it